kairon/chat/routers/web_client.py

```
@@ -34,6 +37,26 @@ async def get_agent_response_for_web_client(request: ChatRequest,
                                     current_user.is_integration_user, request.metadata)
     return {"data": response}
 
+@router.post("/chat/media", response_model=Response)
+async def get_agent_response_for_web_client2(
+                                            data: str = Form(...),
+                                            metadata: str = Form(...),
+                                            files: list[UploadFile] = File(...),
```

Ruff (0.8.2)

Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)